library/cohesity_job.py

```diff
@@ -428,6 +428,32 @@ def get_vmware_vm_ids(module, job_meta_data, job_details, vm_names):
         raise__cohesity_exception__handler(error, module)
 
 
+def get_view_storage_domain_id(module, self):
+    '''
+    function to get view's storage domain id.
+    :param module:
+    :param self:
+    :return:
+    '''
+    server = module.params.get('cluster')
+    validate_certs = module.params.get('validate_certs')
+    token = self['token']
+    view_name = module.params.get('view_name')
+    try:
+        uri = "https://" + server + "/irisservices/api/v1/public/views/" + view_name
```

Comment:
Use SDK ?

Reply:
the module is an existing one which uses raw API calls, if we use SDK here, it add more code plus it would be a mix of sdk and raws api calls. I am using sdk for any new modules we are adding.

Reply:
Ack , ok